encuentro/data.py

```diff
@@ -294,15 +292,15 @@ def __setitem__(self, pos, value):
 
     def values(self):
         """Return the iter values of the data."""
-        return self.data.itervalues()
+        return iter(self.data.values())
```

Comment:
IIRC, no need to do the `iter()` here and in the other cases, it was just an optimization which is the default in Py3.